bundles/tools.vitruv.applications.umljava/src/tools/vitruv/applications/umljava/uml2java/UmlToJavaTag.java

```diff
@@ -0,0 +1,10 @@
+package tools.vitruv.applications.umljava.uml2java;
+
+public final class UmlToJavaTag {
+	public static final String GETTER = "getter";
+	public static final String SETTER = "setter";
+	
+	public static final String[] ACCESSORS = {GETTER, SETTER};
+	
+	private UmlToJavaTag() { }
+}
```

Comment:
This is perfectly fine as it is, however, I like to use [`@Utility`](https://github.com/kit-sdq/XAnnotations#utility) annotation for such classes. It is a bit of syntactic sugar which makes utility classes explicit. The dependency should probably already be there and then we can remove the constructor and the final keyword.